test_resources/user_key_value_types/meow.edn

```diff
@@ -0,0 +1,4 @@
+[:map
```

Comment:
```suggestion
;; this is solely used in tests
[:map
```

Reply:
actually I think it's OK to remove this comment from `cats.edn` since these are clearly in the `test_resources` directory